linters/module-2/.eslintrc

```diff
@@ -0,0 +1,86 @@
+{
+  "parser": "babel-eslint",
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended"
+  ],
+  "plugins": [
+    "react"
+  ],
+  "env": {
+    "browser": true,
+    "mocha": true,
+    "node": true,
+    "es6": true
+  },
+  // Having a problem with one of these rules? Learn more about it here: https://eslint.org/docs/rules/
```

Comment:
Sorry, this needs to be removed from ours. Will you kill this line?

Reply:
No need to be sorry! Done!